api/observationunits.py

```diff
@@ -31,7 +32,7 @@ def search(germplasmDbId=None, observationVariableDbId=None, studyDbId=None, loc
                     t.id::text as observationDbId, \
                     t.mean::text as value, \
                     t.date as observationTimeStamp, \
-                    s.sitename as location_abbreviation, \
+                    s.sitename as observationUnitName, \
```

Comment:
do we want to get the observationUnitId (siteid) as well if that is part of the spec.